src/signer_node/message_processor/process_blockparticipants.rs

```diff
@@ -469,16 +456,21 @@ mod tests {
 
         let expected_localsig = to_local_sig(&v["expected_localsig"]);
         let expected_block_shared_keys = to_block_shared_keys(&v["expected_block_shared_keys"]);
+        let expected_participants: HashSet<SignerID> = {
```

Comment:
These changes for adding "expected_participants" parameter is required to this Pull Request?

Reply:
removed.